src/components/Modals/Onboarding/Onboarding.vue

```diff
@@ -185,11 +188,11 @@ export default defineComponent({
       if (this.name.lastName === '') {
         messages.push('a last name');
       }
-      if (this.onboarding.gradYear === '') {
-        messages.push('a graduation year');
+      if (this.onboarding.gradYear === '' || !this.onboarding.gradSem) {
+        messages.push('a graduation date');
```

Comment:
I wrote it this way intentionally because it will lead to a shorter error message than if we list ("entrance season, entrance year, graduation season, graduation year, and ...") while still remaining relatively clear. I also think the phrase "Entrance Season" or "Graduation Season" aren't especially clear or used by non cp-devs. Let me know if you still think it is unclear and I can change!

Reply:
that makes sense! in that case, could we just say graduation semester to encompass both the year and season?

Reply:
Let me check with a designer! Will go with what they prefer since I made this up